client/src/main/java/org/spine3/base/StringifierRegistry.java

```diff
@@ -58,17 +59,27 @@ private StringifierRegistry() {
         // Prevent external instantiation of this singleton class.
     }
 
+    @SuppressWarnings("unchecked") // It is OK because the class is checked before the cast.
```

Comment:
Can we extract the bit, where the unsafe cast is made, to a separate method and move the suppression to it?